src/Console/Installer.php

```diff
@@ -156,7 +156,17 @@ public static function setFolderPermissions($dir, $io)
             }
         };
 
-        $worldWritable = bindec('0000000111');
+        $worldWritable = 007;
+        if ($worldWritable === 007) {
+            $io->write(
+                'NOTICE: "./tmp" and "./logs" permissions are set to world read- and writable. '
+                . 'Depending on your requirements, you may want to change this before in '
+                . __FILE__ . ' on line #' . (__LINE__ - 5) . ' before production deployment '
```

Comment:
Are you really suggesting that users edit this file?

How does a developer creating a new project have anything to do with deployment? :confused: